omnibus/Gemfile

```diff
@@ -6,14 +6,15 @@ gem "artifactory"
 
 gem "pedump"
 
+# Use Berkshelf for resolving cookbook dependencies
+# Using a branch here that calls out a berkshelf version for chef-17
+gem "berkshelf", git: "https://github.com/chef/berkshelf.git", branch: "jfm/chef17_berkshelf_tweak"
```

Comment:
Will this branch not be released?

Reply:
I am guessing it might not since the Chef/Berkshelf repo is a fork from Berkshelf/Berkshelf

Reply:
What issue do we currently have with using `gem "berkshelf", ">= 7.0"` ?

Reply:
It was always pulling in Chef-18 and causing build/test failures. I needed a way to get a chef-17 dependency in the mix too.

Reply:
@ramereth I was wrong. I COULD merge that in. Now I just need to redo the PR.